shopfloor_mobile_base_auth_user/static/wms/src/login.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2021 Camptocamp SA (http://www.camptocamp.com)
+ * @author Simone Orsi <simone.orsi@camptocamp.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+import {translation_registry} from "/shopfloor_mobile_base/static/wms/src/services/translation_registry.js";
+import {
+    AuthHandlerMixin,
+    auth_handler_registry,
+} from "/shopfloor_mobile_base/static/wms/src/services/auth_handler_registry.js";
+
+// Provide auth handle for Odoo calls
+export class UserAuthHandler extends AuthHandlerMixin {
+    // Not sure we need anything if we get a cookie in the same browser
+    get_params($root) {
+        return {
+            /**
+             * NOTE: we don't have to provide any param because the auth
+             * comes from the cookie.
+             * In the future if we want to support cross domain
+             * we'll have to provide `credentials` same-origin|include
+             * to the fetch method.
+             * */
+
+            headers: {},
+        };
+    }
+
+    on_login($root, evt, data) {
+        evt.preventDefault();
```

Comment:
```suggestion
    on_login($root, data) {
```
`evt.preventDefault();` already done in parent